azure-pipelines.yml

```diff
@@ -198,13 +208,15 @@ jobs:
     displayName: Build_Windows_x64
     publishRidAgnosticPackages: true
     targetArchitecture: x64
+    timeoutInMinutes: 120
 
 # Windows x86
 - template: /eng/jobs/windows-build.yml
   parameters:
     buildFullPlatformManifest: true
     displayName: Build_Windows_x86
     targetArchitecture: x86
+    timeoutInMinutes: 120
```

Comment:
Given that you set the default value of the parameter in the templates, you shouldn't need to pass down the value unless you want to override it to a different value from 120